guides/common/modules/con_configuring-host-collections.adoc

```diff
@@ -1,8 +1,7 @@
 [id="Configuring_Host_Collections_{context}"]
 = Configuring Host Collections
 
-ifdef::foreman-deb,foreman-el,katello[]
-This is for users of the Katello plug-in and hosts running RPM-based linux distributions.
+ifdef::katello[]
 Hosts collections work via the Pulp back end.
```

Comment:
Does this sentence add any value for end users here? I think line 8++ is a more user-friendly description. What do you think about dropping lines 4-6?

Reply:
I had doubts about it. It may even be a detractor and lowering the value: it's confusing to me. But I wanted to limit the scope of this PR.

What do others think?

Reply:
I think you can go ahead and remove it.

Reply:
Removed